templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,31 @@
+# Configure authentication methods
+
+You can configure your organization settings to allow users to log in using
+different methods. Users may already have accounts with some of the
+authentication methods and others can be integrated with existing systems.
+
+{!admin-only.md!}
+
+{!go-to-the.md!} [Authentication methods](/#organization/auth-methods)
+{!admin.md!}
+
+2. Toggle the checkboxes next to the following options to configure your organization's authentication methods:
+
+     * **Email** - Use an email and password created on Zulip to log in.
+
+     * **GitHub** - Use [GitHub](https://github.com/) accounts to log in.
```

Comment:
You may want to talk about OAuth2 applications for both of the GitHub and Google methods.